saleor/settings.py

```diff
@@ -470,6 +471,11 @@ def get_host():
     "saleor.core.auth_backend.PluginBackend",
 ]
 
+# Expired checkouts settings - defines after what time checkouts will be deleted
+ANONYMOUS_CHECKOUTS_TIMEDELTA = {"days": 30}
+USER_CHECKOUTS_TIMEDELTA = {"days": 90}
+EMPTY_CHECKOUTS_TIMEDELTA = {"hours": 6}
```

Comment:
In other time delta settings we use:
`JWT_TTL_REFRESH = timedelta(seconds=parse(os.environ.get("JWT_TTL_REFRESH", "30 days")))`
In my opinion, we should store it in the same way.